interactions/api/models/guild.py

```diff
@@ -768,6 +768,25 @@ async def create_channel(
 
         return Channel(**res, _client=self._client)
 
+    async def clone_channel(self, channel_id: int) -> Channel:
+        """
+        Clones a channel of the guild.
+
+        :param channel_id: The id of the channel to clone
+        :type channel_id: int
+        :return: The created (cloned) channel
```

Comment:
Probably just say cloned for consistency
```suggestion
        :return: The cloned channel
```